weave/integrations/openai/openai_sdk.py

```diff
@@ -10,6 +10,38 @@
     from openai.types.chat import ChatCompletionChunk
 
 
+def maybe_unwrap_api_response(value: typing.Any) -> typing.Any:
```

Comment:
So we need this because LiteLLM is using the wrapped version for some reason?

Any idea why they do that?

Reply:
That is exactly why. They are requesting the raw responses and then consuming the headers